js.go

```diff
@@ -267,16 +267,16 @@ const (
 	TraceReceived TraceOperation = 1
 )
 
-// TraceCB is called to trace API interactions for the JetStream Context
-type TraceCB func(op TraceOperation, subj string, payload []byte, hdr Header)
```

Comment:
If we remove this, then why not remove TraceOperation (with TraceSent and TraceReceived) above?

Reply:
Yes can be removed, we haven't tagged a release of the client that included these so should be ok.

Reply:
Oh wow... we are due for a release soon then ;-). But then that's good, so it is a bit less of an issue then.